efi/generate_binary.py

```diff
@@ -73,8 +73,19 @@ def _add_nx_pefile(args):
     try:
         import pefile
     except ImportError:
-        print("Unable to add NX support to binaries without genpeimg or python3-pefile")
-        sys.exit(1)
+        print("Unable to add NX support to binaries without genpeimg or python3-pefile, adding NX support manually")
```

Comment:
It's no longer true that it can't be done. You're doing it!

Reply:
I'll rephrase it :)